MekHQ/src/mekhq/campaign/parts/MissingAvionics.java

```diff
@@ -60,8 +60,12 @@ public int getBaseTime() {
                 time = 600;
             }
             return time;
+	// CamOps, 3rd printing, page 207: 80 hours for Large Craft, 8 hours otherwise
```

Comment:
I feel like we should just remove that option and go with the latest printing at this point.

Reply:
Yeah, that's basically what I'm getting at.

Reply:
Could we remove the option in a different pull request? I'm not familiar with everything that has to be changed, and it could break saved presets in a way this fix won't.

Reply:
Sure, why not.